### DISCUSSION TITLE
Run admin in development and connect to running hetty

### DISCUSSION THREAD

**Discussion by MichalZalecki** [dstotijn/hetty#47] [Q&A] [answered]
Is there any other way to develop admin with `cmd/hetty/main.go` running that doesn't require building the frontend each time and using `yarn dev` instead?

**Comment by dstotijn** [comment, marked as answer]
Yes, here's what I do when I'm developing:
* Run `modd` from the root directory of the repository, for running `hetty` on `:8080` with file watching of `*.go` files
* Run `yarn dev` from the `admin` directory. This runs Next.js in dev mode with file watching/hot reloading. It runs on http://localhost:3000, and proxies API reqs to the Go server running on `:8080` ([src](https://github.com/dstotijn/hetty/blob/master/admin/next.config.js#L6-L13))

Then I visit http://localhost:3000 and use that as I code.

**Reply by MichalZalecki** [reply]
Thank you. I'll try that!
